scalableminds/webknossos
# Remove dataset from thumbnail cache

### URL of deployed dev instance (used for testing):
- https://removefromthumbnailcache.webknossos.xyz

### Steps to test:
- new route `PUT /datasets/:organizationName/:dataSetName/clearThumbnailCache` (no body expected)
- (should be triggered by the reload button in the dataset list view)
- after that, a hard page refresh in the gallery should trigger new thumbnail requests from the wK backend to the datastore

### Issues:
- fixes #3808

------
